### DISCUSSION TITLE
List of items as in Quasar Qlist

### DISCUSSION THREAD

**Discussion by meslahik** [zauberzeug/nicegui#1693] [Ideas / Feature Requests]
Currently, NiceGUI does not have a component for list of items. It is possible to create list of items through combining available components (as it is done with sidebar in [NiceGUI's documentation](https://nicegui.io/documentation) ). However, it would be great to have the feature-rich component of Quasar Qlist.

The component might be used like `ui.list()`

**Comment by falkoschindler** [comment]
I agree!
Just for reference, there's already another similar feature request over here: https://github.com/zauberzeug/nicegui/discussions/512.
